card/mod/02_basic_types/spec/set/all/rss_spec.rb

```diff
@@ -9,7 +9,13 @@
   it 'should handle faulty search queries' do
     bad_search = Card.create! :name=>'Bad Search', :type=>'Search', :content=>'not no search'
     rendered = bad_search.format(:rss).render_feed_body
-    expect(rendered).to match(/item/)
+    expect(rendered).to have_tag("item",:text=>/JSON::ParserError/)
+  end
+
+  it 'should handle bad queries' do
+    bad_search = Card.create! :name=>'Bad Search', :type=>'Search', :content=>'{"found_by":"home"}'
```

Comment:
Line is too long. [99/80]<br>Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator '=>'.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.